src/introduction/what_data.md

```diff
@@ -0,0 +1,44 @@
+# What Data does Mozilla Collect?
+
+The data Mozilla collects can roughly be categorized into three categories: product telemetry, usage logs and website telemetry.
+
+## Product Telemetry
+
+Most of our products, including Firefox, are instrumented to send small JSON packets called "pings" when telemetry is enabled.
+Pings include some combination of environment data (e.g., information about operating system and hardware), measurements (e.g., for Firefox, information about the maximum number of open tabs and time spent running in JavaScript garbage collections), and events (indications that something has happened).
+
+Inside Firefox, most Telemetry is collected via a module called "Telemetry".
+The details of our ping formats is extensively documented in the Firefox Source Docs under [Toolkit/Telemetry].
+
+In newer products like Firefox for Android, instrumentation is handled by the [Glean SDK], whose design was inspired from what Mozilla learned from the implementation of the Telemetry module and has many benefits.
+At some point in the near future, Mozilla plans to replace the Telemetry module with the Glean SDK.
+For more information, see [Firefox on Glean (FOG)].
+
+[glean sdk]: ../concepts/glean/glean.md
+[toolkit/telemetry]: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/index.html
+[firefox on glean (fog)]: https://firefox-source-docs.mozilla.org/toolkit/components/glean/index.html
+
+When submissions from our clients hit our end points, ping data is aggregated into ping datasets.
+On a daily basis, these ping-level datasets are rolled into derived datasets which are easier to reason about and faster to query.
+You can learn more about this in [Guiding Principles for Data Infrastructure].
+
+Both the ping and derived datasets are viewable using tools like GLAM and Looker.
```

Comment:
Can we get a link to GLAM? I don't know what GLAM is

Reply:
Just learned about GLAM as I continued reading the intro. It's amazing!

Reply:
Yay! For sure I'll add a link, I think I put them in the dictionary documentation already.